clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -376,6 +377,10 @@ public static Properties addDeserializerToConfig(Properties properties,
         super(CONFIG, props);
     }
 
+    public static Map<String, ConfigKey> configKeys() {
```

Comment:
Should this be `configNames` instead with an implementation like:

```java
CONFIG.names()
```

It doesn't seem like we actually need `ConfigKey`.